tahaafzal5/AugieGeoTag
# Test Tool Complete

Test Tool Complete:
1. add verify, print, and print tag
2. change error prompt to err
3. some annotation added
